runtime/caml/io.h

```diff
@@ -75,18 +77,26 @@ CAMLextern file_offset caml_pos_out (struct channel *);
 /* I/O on channels from C. The channel must be locked (see below) before
    calling any of the functions and macros below */
 
+#define caml_putch(channel, ch) do{                                       \
+  if ((channel)->curr >= (channel)->end) caml_flush_partial(channel);     \
+  *((channel)->curr)++ = (ch);                                            \
+}while(0)
+
+#define caml_getch(channel)                                                 \
+  ((channel)->curr >= (channel)->max                                        \
+   ? caml_refill(channel)                                                   \
+   : (unsigned char) *((channel)->curr)++)
+
```

Comment:
On trunk these macros were recently turned into regular functions.

Reply:
Yes, something got lost here. 
Fixed in 2ccf7ce